README.md

```diff
@@ -147,9 +126,22 @@ Add the following line to your `~/.config/fish/config.fish`:
 zoxide init fish | source
 ```
 
-<!-- omit in toc -->
+#### nushell
 
-#### PowerShell
+Initialize zoxide's Nushell script:
+
+```sh
+zoxide init nushell --hook prompt | save ~/.zoxide.nu
+```
+
+Then, in your Nushell configuration file:
+
+- Prepend `__zoxide_hook|` to your `prompt` variable.
```

Comment:
Is there a specific reason for the `|` after `__zoxide_hook`.
Is there a specific reason that the prompt shouldn't be prepended with `__zoxide_hook ;`

Reply:
Fixed, thanks!